components/mintKamon/GenerateImageForm.tsx

```diff
@@ -1,13 +1,96 @@
-import { Button, Heading } from "@chakra-ui/react"
-import { Layout } from "@/components/layouts/layout"
+import { useAccount } from 'wagmi'
+import { useState } from 'react'
+import { Button, Heading, Text, FormControl, FormLabel, Input } from '@chakra-ui/react'
+import { Layout } from '@/components/layouts/layout'
 
 const GenerateImageForm = () => {
+  const ipfsApiEndpoint = 'https://api.staging.sakazuki.xyz/henkaku/ipfs'
```

Comment:
If the URL is likely to change depending on the environment, it may be better to read the URL using environment variables.

Note that the environment variable is special if you want to view it from a browser with Next.js

https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser